app/assess/routes.py

```diff
@@ -54,6 +61,21 @@ def display_sub_criteria(
                 sub_criteria_id=sub_criteria_id,
             )
             scores_submitted = True
+     
+        elif commentForm.validate_on_submit():
+            comment=commentForm.comment.data
+            user_id = g.account_id
+            theme_id=request.args['theme_id']
+            displayCommentBox = False
+
+            submit_comment(
+                    comment=comment,
+                    application_id=application_id,
+                    sub_criteria_id=sub_criteria_id,
+                    user_id=user_id,
+                    theme_id=theme_id
+                )
+            return redirect(request.path + "?theme_id=" + theme_id)
```

Comment:
Could probably use flasks built-in `url_for` for this as opposed to adding the query parameter via string concatenation